src/poliastro/constants/rotational_elements.py

```diff
@@ -0,0 +1,209 @@
+from astropy.constants import Constant
+
+T_earth = Constant(
+    "T_earth",
+    "Earth rotational period",
+    0.9972698,
+    "d",
+    0,
+    "Vallado, D. and McClain, W. Fundamentals of astrodynamics and applications.",
+    system="si",
+)
+
+T_sun = Constant(
+    "T_sun",
+    "Sun rotational period",
+    25.38,
+    "d",
+    0,
+    "Vallado, D. and McClain, W. Fundamentals of astrodynamics and applications.",
+    system="si",
+)
+
+T_mercury = Constant(
+    "T_mercury",
+    "Mercury rotational period",
+    58.6462,
+    "d",
+    0,
+    "Vallado, D. and McClain, W. Fundamentals of astrodynamics and applications.",
+    system="si",
+)
+
+T_venus = Constant(
+    "T_venus", "Venus rotational period", -243.01, "d", 0, "", system="si"
+)
+
+T_moon = Constant(
+    "T_moon",
+    "Moon rotational period",
+    27.32166,
+    "d",
+    0,
+    "Vallado, D. and McClain, W. Fundamentals of astrodynamics and applications.",
+    system="si",
+)
+
+T_mars = Constant(
+    "T_mars",
+    "Mars rotational period",
+    1.02595675,
+    "d",
+    0,
+    "Vallado, D. and McClain, W. Fundamentals of astrodynamics and applications.",
+    system="si",
+)
+
+T_jupiter = Constant(
+    "T_jupiter",
+    "Jupiter rotational period",
+    0.41354,
+    "d",
+    0,
+    "Vallado, D. and McClain, W. Fundamentals of astrodynamics and applications.",
+    system="si",
+)
+
+T_saturn = Constant(
+    "T_saturn",
+    "Saturn rotational period",
+    0.4375,
+    "d",
+    0,
+    "Vallado, D. and McClain, W. Fundamentals of astrodynamics and applications.",
+    system="si",
+)
+
+T_uranus = Constant(
+    "T_uranus",
+    "Uranus rotational period",
+    -0.65,
+    "d",
+    0,
+    "Vallado, D. and McClain, W. Fundamentals of astrodynamics and applications.",
+    system="si",
+)
+
+T_neptune = Constant(
+    "T_neptune", "Neptune rotational period", 0.768, "d", 0, "", system="si"
+)
+
+T_pluto = Constant(
+    "T_pluto",
+    "Pluto rotational period",
+    -6.3867,
+    "d",
+    0,
+    "Vallado, D. and McClain, W. Fundamentals of astrodynamics and applications.",
+    system="si",
+)
+
+angular_speed_earth = Constant(
+    "angular_speed_earth",
+    "Earth angular speed",
+    7.29211502e-05,
+    "rad / s",
+    0,
+    "Vallado, D. and McClain, W. Fundamentals of astrodynamics and applications.",
+    system="si",
+)
+
+angular_speed_sun = Constant(
```

Comment:
I propose we use `omega_Sun`, `omega_Earth`, etc

Reply:
Also, aren't these definitions redundant with the rotational periods? I think we should only keep the angular velocities

Reply:
Also, if you could also point out where are these values defined?

Reply:
What we discussed in private is that this collisions with how the WGCRE defines the rotational velocities. It's nice to have mean values though, so we will probably want to keep both...?

Reply:
@Juanlu001 This values weren't explicitly defined in Vallado and they were derived from the rotational period reference (this is partly why I included both instead of only the rotational period). I can remove the rotational period and add a docstring explaining this if it'd be better

Reply:
Yes, if these are derived values let's not include them in constants, and only the rotational periods. Then we could have a property like `body.angular_speed` that would do the required operation. This way we keep `poliastro.constants` only with stuff we can check in a book, a paper or similar.